mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -3324,6 +3324,34 @@ LogicalResult CallIntrinsicOp::verify() {
   return success();
 }
 
+void CallIntrinsicOp::build(OpBuilder &builder, OperationState &state,
+                            mlir::StringAttr intrin, mlir::ValueRange args) {
+  build(builder, state, /*results=*/Type{}, intrin, args, FastmathFlagsAttr{},
```

Comment:
```suggestion
  build(builder, state, /*results=*/TypeRange{}, intrin, args, FastmathFlagsAttr{},
```
I think this is a type range (probably Type converts to TypeRange)?

Reply:
Thanks, I've addressed that.